### ISSUE TITLE
Copy to Clipboard Toast: keyboard focus order is not logical

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1217]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/79ef89d6-5d25-11ee-87cc-4bed2ffb3a9d?activeTab=dt-issue&page=0&sortField=ordinal&sortDir=asc&row=3&issuesCount=&issuesCount=&filter=

## Description

Keyboard focus order is not logical in such a way that it affects the meaning or operability of the content.

Element description 1: Example copied to clipboard
Element location: On the "Accordion" page

## Steps to reproduce:

- Without dismissing the "example copied to clipboard" status message, trigger the same copy button
- on the next tab focus, the focus goes to the browser window.

Element description 2: Example copied to clipboard dismiss button receives focus but is not visible to the keyboard-only/ visible users.
Element location: On the "Accordion" page

## Steps to reproduce:

- Without dismissing the "example copied to clipboard" status message, trigger another Copy button
   - The same message appears below the existing message. So when navigating using a tab, the dismiss button receives focus but is not visible to the keyboard-only/ visible users.'

Note: It is applicable to all the pages
